jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/test/java/org/jbpm/workbench/pr/backend/server/RemoteProcessRuntimeDataServiceImplTest.java

```diff
@@ -55,4 +91,60 @@ private void assertMethodResult(final Method method, final Class<?> returnType,
         }
     }
 
+    @Test
+    public void testGetProcesses(){
+        final ProcessDefinition def = ProcessDefinition.builder().id("processId").build();
+
+        when(queryServicesClient.findProcesses(0, 0, "", true)).thenReturn(singletonList(def));
+
+        final List<ProcessSummary> summaries = service.getProcesses("serverTemplateId", 0, 0, "", true);
+
+        assertNotNull(summaries);
+        assertEquals(1, summaries.size());
+        assertProcessSummary(def, summaries.get(0));
+    }
+
+    @Test
+    public void testGetProcessesByFilter(){
+        final ProcessDefinition def = ProcessDefinition.builder().id("processId").build();
+
+        when(queryServicesClient.findProcesses("filter", 0, 0, "", true)).thenReturn(singletonList(def));
+
+        final List<ProcessSummary> summaries = service.getProcessesByFilter("serverTemplateId", "filter", 0, 0, "", true);
+
+        assertNotNull(summaries);
+        assertEquals(1, summaries.size());
+        assertProcessSummary(def, summaries.get(0));
+    }
+
+    @Test
+    public void testGetProcessesByContainerIdProcessId(){
+        final String processId = "processId";
+        final String containerId = "containerId";
```

Comment:
Since some combination of processId, containerId and serverTemplateId is needed in every test, maybe makes sense to make class fields out of them and use them directly in tests.